uctMaths/competition/compadmin.py

```diff
@@ -630,6 +634,51 @@ def export_awards(request):
     output_workbook.save(response)
     return response
 
+def makeCertificates(students, assigned_school):
+    
+    certPath = "../../Certificates/"
+
+    if len(students) > 0:
+        awardCerts = {"G": "goldTemplate.docx", "M": "meritTemplate.docx", None: "participationTemplate.docx", "MOX": "meritOxfordTemplate.docx", "OX": "oxfordTemplate"}
+
+        try:
+            os.mkdir(str(students[0].school).replace(" ", "_") + "_Certificates")
```

Comment:
Hmm, I'm not sure how good of an idea it is to copy this file into the same folder we run code from. I think we should probably create a temporary directory for this with `tempfile.mkdtemp`